src/rpm/builder.rs

```diff
@@ -306,6 +306,50 @@ impl PackageBuilder {
         Ok(self)
     }
 
+    /// Add multiple file to the package.
+    ///
+    /// ```
+    /// # fn foo() -> Result<(), Box<dyn std::error::Error>> {
+    /// let files =vec![(
+    ///         "./awesome-config.toml",
+    ///         rpm::FileOptions::new("/etc/awesome/config.toml").is_config(),
+    ///     ),    ///     // file mode is inherited from source file
+    ///     (
+    ///         "./awesome-bin",
+    ///         rpm::FileOptions::new("/usr/bin/awesome"),
+    ///     ),
+    ///     (
+    ///         "./awesome-config.toml",
+    ///         // you can set a custom mode, capabilities and custom user too
+    ///         rpm::FileOptions::new("/etc/awesome/second.toml").mode(0o100744).caps("cap_sys_admin=pe")?.user("hugo"),
+    ///     )];
+    /// let pkg = rpm::PackageBuilder::new("foo", "1.0.0", "Apache-2.0", "x86_64", "some baz package")
+    ///     .with_files(files)?
+    ///     .build()?;
+    /// # Ok(())
+    /// # }
+    /// ```
+
+    pub fn with_files(
+        mut self,
+        files: Vec<(impl AsRef<Path>, impl Into<FileOptions>)>,
```

Comment:
Could we make these explicit generics? It allows the user to specify the types in case of ambiguities.

Reply:
I am not a particular fan of tuples.